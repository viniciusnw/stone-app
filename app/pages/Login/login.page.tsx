import React from "react";
import { View } from "react-native";
import { LoginBox } from "@Components";
import styled from "styled-components";

const StyledWrapper = styled(View)`
  width: 100%;
  padding: 20px;
  align-items: center;
  justify-content: center;
`;

export default function Login(props: any) {
  const loginBoxProps = {
    facebook: {
      onPress: () => props.goTo('Home'),
    },
    google: {
      onPress: () => props.goTo('Home'),
    },
    email: {
      value: "bruce@wayne.corp",
      onChangeText: () => true,
      placeholder: "E-mail",
      autoCompleteType: "email",
    },
    password: {
      value: "123456",
      onChangeText: () => true,
      placeholder: "Senha",
      autoCompleteType: "password",
      secureTextEntry: true
    },
    forgotPassword: true,
    submit: () => props.goTo('Home'),
    register: {
      onPress: () => true,
    },
  };

  return (
    <StyledWrapper>
      <View style={{ width: "100%", marginTop: 60 }}>
        <LoginBox {...loginBoxProps} />
      </View>
    </StyledWrapper>
  );
}
