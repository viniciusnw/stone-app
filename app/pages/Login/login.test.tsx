import React from "react";
import renderer from "react-test-renderer";
import { lightTheme } from "@Theme";
import { ThemeProvider } from "styled-components/native";
import Login from "./login.page";
import "jest-styled-components";
import Wrapper from "@/app/navigation/wrapper.navigator";

jest.useFakeTimers();

jest.mock("@react-navigation/native", () => {
  return {
    useNavigation: () => ({
      navigate: jest.fn(),
      dispatch: jest.fn(),
    }),
    useRoute: () => ({
      params: {
        productId: 1,
      },
    }),
  };
});

test("LoginWrapper", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={lightTheme}>
        <Wrapper Page={Login} header bottomBarType={null} />
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test("Login", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={lightTheme}>
        <Login setState={jest.fn} />
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
