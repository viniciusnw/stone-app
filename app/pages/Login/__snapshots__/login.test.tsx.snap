// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login 1`] = `
<View
  className=""
>
  <View
    style={
      {
        "marginTop": 60,
        "width": "100%",
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "justifyContent": "center",
          "width": "100%",
        }
      }
    >
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            {
              "fontFamily": "FredokaOne-Regular",
            }
          }
        >
          <Text
            style={
              {
                "color": "rgb(0,168,104)",
                "fontSize": 32,
                "fontWeight": "700",
              }
            }
          >
            Stone
          </Text>
          <Text
            style={
              {
                "color": "rgb(10,245,61)",
                "fontSize": 32,
                "fontWeight": "700",
              }
            }
          >
            Ton :)
          </Text>
        </Text>
      </View>
      <View
        style={
          {
            "borderColor": "#ddd",
            "borderStyle": "solid",
            "borderWidth": 1,
            "marginTop": 30,
            "width": "100%",
          }
        }
      >
        <Text
          style={
            {
              "fontSize": 26,
              "fontWeight": "700",
              "marginBottom": 25,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 20,
              "textAlign": "center",
            }
          }
        >
          Login
        </Text>
        <View
          style={
            {
              "flexDirection": "row",
              "width": "100%",
            }
          }
        >
          <View
            style={
              [
                {
                  "flexBasis": "50%",
                  "paddingBottom": 0,
                  "paddingLeft": 10,
                  "paddingRight": 10,
                  "paddingTop": 0,
                },
                {
                  "paddingLeft": 20,
                },
              ]
            }
          >
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                {
                  "alignItems": "center",
                  "backgroundColor": "#111111",
                  "borderBottomLeftRadius": 0,
                  "borderBottomRightRadius": 0,
                  "borderTopLeftRadius": 0,
                  "borderTopRightRadius": 0,
                  "display": "flex",
                  "height": 40,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 10,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#ffffff",
                    "fontSize": 14,
                    "fontWeight": "700",
                  }
                }
              >
                Facebook
              </Text>
            </View>
          </View>
          <View
            style={
              [
                {
                  "flexBasis": "50%",
                  "paddingBottom": 0,
                  "paddingLeft": 10,
                  "paddingRight": 10,
                  "paddingTop": 0,
                },
                {
                  "paddingRight": 20,
                },
              ]
            }
          >
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                {
                  "alignItems": "center",
                  "backgroundColor": "#111111",
                  "borderBottomLeftRadius": 0,
                  "borderBottomRightRadius": 0,
                  "borderTopLeftRadius": 0,
                  "borderTopRightRadius": 0,
                  "display": "flex",
                  "height": 40,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 10,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#ffffff",
                    "fontSize": 14,
                    "fontWeight": "700",
                  }
                }
              >
                Google
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            {
              "alignItems": "center",
              "borderColor": "transparent",
              "borderStyle": "solid",
              "borderTopColor": "#ddd",
              "borderWidth": 1,
              "marginBottom": 20,
              "marginLeft": 20,
              "marginRight": 20,
              "marginTop": 30,
            }
          }
        >
          <Text
            style={
              {
                "backgroundColor": "#ffffff",
                "height": 20,
                "marginTop": -10,
                "textAlign": "center",
                "width": 40,
              }
            }
          >
            ou
          </Text>
        </View>
        <View
          style={
            {
              "paddingBottom": 20,
              "paddingLeft": 20,
              "paddingRight": 20,
              "paddingTop": 0,
            }
          }
        >
          <View
            style={
              {
                "marginBottom": 15,
              }
            }
          >
            <TextInput
              autoCompleteType="email"
              onChangeText={[Function]}
              placeholder="E-mail"
              style={
                {
                  "borderColor": "#ddd",
                  "borderStyle": "solid",
                  "borderWidth": 1,
                  "height": 42,
                  "paddingBottom": 10,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 10,
                  "width": "100%",
                }
              }
              value="bruce@wayne.corp"
            />
          </View>
          <View
            style={
              {
                "marginBottom": 15,
              }
            }
          >
            <TextInput
              autoCompleteType="password"
              onChangeText={[Function]}
              placeholder="Senha"
              secureTextEntry={true}
              style={
                {
                  "borderColor": "#ddd",
                  "borderStyle": "solid",
                  "borderWidth": 1,
                  "height": 42,
                  "paddingBottom": 10,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 10,
                  "width": "100%",
                }
              }
              value="123456"
            />
          </View>
          <View
            style={
              {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                {
                  "alignItems": "center",
                  "backgroundColor": "rgb(0,168,104)",
                  "borderBottomLeftRadius": 0,
                  "borderBottomRightRadius": 0,
                  "borderTopLeftRadius": 0,
                  "borderTopRightRadius": 0,
                  "display": "flex",
                  "height": 40,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 10,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#ffffff",
                    "fontSize": 14,
                    "fontWeight": "700",
                  }
                }
              >
                Entrar
              </Text>
            </View>
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={false}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                {
                  "opacity": 1,
                }
              }
            >
              <Text>
                Recuperar senha
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          {
            "alignItems": "center",
            "marginTop": 30,
          }
        }
      >
        <Text
          style={
            {
              "marginBottom": 15,
            }
          }
        >
          Ainda não possui uma conta?
        </Text>
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "backgroundColor": "rgb(0,168,104)",
              "borderBottomLeftRadius": 0,
              "borderBottomRightRadius": 0,
              "borderTopLeftRadius": 0,
              "borderTopRightRadius": 0,
              "display": "flex",
              "height": 40,
              "justifyContent": "center",
              "opacity": 1,
              "paddingBottom": 10,
              "paddingLeft": 15,
              "paddingRight": 15,
              "paddingTop": 10,
            }
          }
        >
          <Text
            style={
              {
                "color": "#ffffff",
                "fontSize": 14,
                "fontWeight": "700",
              }
            }
          >
            Cadastre-se
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`LoginWrapper 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "borderBottomColor": "#ddd",
        "borderColor": "transparent",
        "borderStyle": "solid",
        "borderWidth": 1,
        "flexDirection": "row",
        "height": 62,
        "justifyContent": "space-between",
        "paddingBottom": 0,
        "paddingLeft": 15,
        "paddingRight": 15,
        "paddingTop": 0,
        "width": "100%",
      }
    }
  >
    <View
      style={
        {
          "width": 30,
        }
      }
    />
    <Text
      numberOfLines={2}
      style={
        {
          "color": "#111111",
          "fontSize": 20,
          "fontWeight": "700",
          "maxWidth": 200,
          "textAlign": "center",
        }
      }
    >
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            {
              "fontFamily": "FredokaOne-Regular",
            }
          }
        >
          <Text
            style={
              {
                "color": "rgb(0,168,104)",
                "fontSize": 32,
                "fontWeight": "700",
              }
            }
          >
            Stone
          </Text>
          <Text
            style={
              {
                "color": "rgb(10,245,61)",
                "fontSize": 32,
                "fontWeight": "700",
              }
            }
          >
            Ton :)
          </Text>
        </Text>
      </View>
    </Text>
    <View
      style={
        {
          "width": 30,
        }
      }
    />
  </View>
  <View
    className=""
  >
    <View
      style={
        {
          "marginTop": 60,
          "width": "100%",
        }
      }
    >
      <View
        style={
          {
            "alignItems": "center",
            "justifyContent": "center",
            "width": "100%",
          }
        }
      >
        <View
          style={
            {
              "flexDirection": "row",
            }
          }
        >
          <Text
            style={
              {
                "fontFamily": "FredokaOne-Regular",
              }
            }
          >
            <Text
              style={
                {
                  "color": "rgb(0,168,104)",
                  "fontSize": 32,
                  "fontWeight": "700",
                }
              }
            >
              Stone
            </Text>
            <Text
              style={
                {
                  "color": "rgb(10,245,61)",
                  "fontSize": 32,
                  "fontWeight": "700",
                }
              }
            >
              Ton :)
            </Text>
          </Text>
        </View>
        <View
          style={
            {
              "borderColor": "#ddd",
              "borderStyle": "solid",
              "borderWidth": 1,
              "marginTop": 30,
              "width": "100%",
            }
          }
        >
          <Text
            style={
              {
                "fontSize": 26,
                "fontWeight": "700",
                "marginBottom": 25,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 20,
                "textAlign": "center",
              }
            }
          >
            Login
          </Text>
          <View
            style={
              {
                "flexDirection": "row",
                "width": "100%",
              }
            }
          >
            <View
              style={
                [
                  {
                    "flexBasis": "50%",
                    "paddingBottom": 0,
                    "paddingLeft": 10,
                    "paddingRight": 10,
                    "paddingTop": 0,
                  },
                  {
                    "paddingLeft": 20,
                  },
                ]
              }
            >
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessibilityValue={
                  {
                    "max": undefined,
                    "min": undefined,
                    "now": undefined,
                    "text": undefined,
                  }
                }
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  {
                    "alignItems": "center",
                    "backgroundColor": "#111111",
                    "borderBottomLeftRadius": 0,
                    "borderBottomRightRadius": 0,
                    "borderTopLeftRadius": 0,
                    "borderTopRightRadius": 0,
                    "display": "flex",
                    "height": 40,
                    "justifyContent": "center",
                    "opacity": 1,
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                  }
                }
              >
                <Text
                  style={
                    {
                      "color": "#ffffff",
                      "fontSize": 14,
                      "fontWeight": "700",
                    }
                  }
                >
                  Facebook
                </Text>
              </View>
            </View>
            <View
              style={
                [
                  {
                    "flexBasis": "50%",
                    "paddingBottom": 0,
                    "paddingLeft": 10,
                    "paddingRight": 10,
                    "paddingTop": 0,
                  },
                  {
                    "paddingRight": 20,
                  },
                ]
              }
            >
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessibilityValue={
                  {
                    "max": undefined,
                    "min": undefined,
                    "now": undefined,
                    "text": undefined,
                  }
                }
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  {
                    "alignItems": "center",
                    "backgroundColor": "#111111",
                    "borderBottomLeftRadius": 0,
                    "borderBottomRightRadius": 0,
                    "borderTopLeftRadius": 0,
                    "borderTopRightRadius": 0,
                    "display": "flex",
                    "height": 40,
                    "justifyContent": "center",
                    "opacity": 1,
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                  }
                }
              >
                <Text
                  style={
                    {
                      "color": "#ffffff",
                      "fontSize": 14,
                      "fontWeight": "700",
                    }
                  }
                >
                  Google
                </Text>
              </View>
            </View>
          </View>
          <View
            style={
              {
                "alignItems": "center",
                "borderColor": "transparent",
                "borderStyle": "solid",
                "borderTopColor": "#ddd",
                "borderWidth": 1,
                "marginBottom": 20,
                "marginLeft": 20,
                "marginRight": 20,
                "marginTop": 30,
              }
            }
          >
            <Text
              style={
                {
                  "backgroundColor": "#ffffff",
                  "height": 20,
                  "marginTop": -10,
                  "textAlign": "center",
                  "width": 40,
                }
              }
            >
              ou
            </Text>
          </View>
          <View
            style={
              {
                "paddingBottom": 20,
                "paddingLeft": 20,
                "paddingRight": 20,
                "paddingTop": 0,
              }
            }
          >
            <View
              style={
                {
                  "marginBottom": 15,
                }
              }
            >
              <TextInput
                autoCompleteType="email"
                onChangeText={[Function]}
                placeholder="E-mail"
                style={
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 1,
                    "height": 42,
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                    "width": "100%",
                  }
                }
                value="bruce@wayne.corp"
              />
            </View>
            <View
              style={
                {
                  "marginBottom": 15,
                }
              }
            >
              <TextInput
                autoCompleteType="password"
                onChangeText={[Function]}
                placeholder="Senha"
                secureTextEntry={true}
                style={
                  {
                    "borderColor": "#ddd",
                    "borderStyle": "solid",
                    "borderWidth": 1,
                    "height": 42,
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                    "width": "100%",
                  }
                }
                value="123456"
              />
            </View>
            <View
              style={
                {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                }
              }
            >
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessibilityValue={
                  {
                    "max": undefined,
                    "min": undefined,
                    "now": undefined,
                    "text": undefined,
                  }
                }
                accessible={true}
                collapsable={false}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  {
                    "alignItems": "center",
                    "backgroundColor": "rgb(0,168,104)",
                    "borderBottomLeftRadius": 0,
                    "borderBottomRightRadius": 0,
                    "borderTopLeftRadius": 0,
                    "borderTopRightRadius": 0,
                    "display": "flex",
                    "height": 40,
                    "justifyContent": "center",
                    "opacity": 1,
                    "paddingBottom": 10,
                    "paddingLeft": 15,
                    "paddingRight": 15,
                    "paddingTop": 10,
                  }
                }
              >
                <Text
                  style={
                    {
                      "color": "#ffffff",
                      "fontSize": 14,
                      "fontWeight": "700",
                    }
                  }
                >
                  Entrar
                </Text>
              </View>
              <View
                accessibilityState={
                  {
                    "busy": undefined,
                    "checked": undefined,
                    "disabled": undefined,
                    "expanded": undefined,
                    "selected": undefined,
                  }
                }
                accessibilityValue={
                  {
                    "max": undefined,
                    "min": undefined,
                    "now": undefined,
                    "text": undefined,
                  }
                }
                accessible={true}
                collapsable={false}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  {
                    "opacity": 1,
                  }
                }
              >
                <Text>
                  Recuperar senha
                </Text>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            {
              "alignItems": "center",
              "marginTop": 30,
            }
          }
        >
          <Text
            style={
              {
                "marginBottom": 15,
              }
            }
          >
            Ainda não possui uma conta?
          </Text>
          <View
            accessibilityState={
              {
                "busy": undefined,
                "checked": undefined,
                "disabled": undefined,
                "expanded": undefined,
                "selected": undefined,
              }
            }
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "alignItems": "center",
                "backgroundColor": "rgb(0,168,104)",
                "borderBottomLeftRadius": 0,
                "borderBottomRightRadius": 0,
                "borderTopLeftRadius": 0,
                "borderTopRightRadius": 0,
                "display": "flex",
                "height": 40,
                "justifyContent": "center",
                "opacity": 1,
                "paddingBottom": 10,
                "paddingLeft": 15,
                "paddingRight": 15,
                "paddingTop": 10,
              }
            }
          >
            <Text
              style={
                {
                  "color": "#ffffff",
                  "fontSize": 14,
                  "fontWeight": "700",
                }
              }
            >
              Cadastre-se
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
